proof/dpolicy/Dpolicy.thy

```diff
@@ -1082,7 +1073,7 @@ lemma state_vrefs_asid_pool_transform_rev:
   "\<lbrakk> einvs s; cdl_asid_table (transform s) (fst (transform_asid asid)) = Some poolcap;
      \<not> is_null_cap poolcap; \<not> is_null_cap pdcap; pdptr = cap_object pdcap;
      opt_cap (cap_object poolcap, snd (transform_asid asid)) (transform s) = Some pdcap \<rbrakk> \<Longrightarrow>
-     (pdptr, VSRef (asid && mask ARM_A.asid_low_bits) (Some AASIDPool), Control)
+     (pdptr, asid && mask ARM_A.asid_low_bits, AASIDPool, Control)
```

Comment:
And also because RISCV64 doesn't have `VSRef`, and it turns out that most of this can be mirrored between architectures if we only record the bits we need instead of reusing `VSRef`

Reply:
Of course, that makes a lot of sense.